File: linearSearch/linearSearch.js
var linearSearch = function (list, target) {
    // should return the position of the target if found else return -1
    for (var index = 0; index < list.length; index++) {
        if (list[index] === target) {
            return target;
        }
    }
    return -1;
};
var verify = function () {
    var theList = [1, 3, 5, 6, 7];
    var target = 7;
    var position = linearSearch(theList, target);
    if (position != -1) {
        console.log("target exists at position ", position);
    }
    else {
        console.log("target does not exisit in the list");
    }
};
verify();

File: linearSearch/linearSearch.py
def linearSearch(list, target):
    """"
        Returns the index position of the target in the list if found else return -1
    """
    for  i in range(len(list)):
        # note that len(list) runs in constant time
        if list[i] == target:
            return i
    return -1

def verify():
    theList = [1,3,5,6,7]
    target = 12
    position = linearSearch(list=theList, target=target)
    if position != -1 :
        print("target exists at position ", position)
    else: 
        print("target is not in the list")

verify()